mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -160,7 +163,10 @@ def flash(self, source, target, method, no_reset):
                 return EXIT_CODE_FILE_COULD_NOT_BE_READ
 
             self.logger.debug("copy finished")
-            sleep(4)
+
+            # expected behaviour after file copied for mount point is to disappear
+            self._check_mount_point_disappear(mount_point)
```

Comment:
this will drop mbeds support for boards which doesn't behaviour like that - mostly for very old daplinks.